yang/frr-isisd.yang

```diff
@@ -1494,14 +1490,15 @@ module frr-isisd {
         container srgb {
           description
             "Global blocks to be advertised.";
-          must "./upper-bound > ./lower-bound";
           leaf lower-bound {
+              must "../upper-bound > .";
```

Comment:
Yes; however, their logic on changing to this, i.e., the node with the constraint carrying the must rather than a NP container -- which is supposed to have no meaning outside of organizing) did make some sense, so I was going to leave it.